server/src/api/graphql/schema.graphql

```diff
@@ -0,0 +1,52 @@
+type Author {
+  id: String
+  name: String
+  biography: String
+  bornDate: String
+  books: [Book]
+  totalBooks: Int
+  createdAt: String
+  updatedAt: String
+}
+
+type Book {
+  id: String
+  title: String
+  description: String
+  publishedDate: String
+  authorId: String
+  author: Author
+  rating: Float
+  createdAt: String
+  updatedAt: String
+}
+
+type Review {
+  id: String
+  bookId: String
+  comment: String
+  rating: Int
+  createdAt: String
+  updatedAt: String
+}
```

Comment:
Maybe we can use `!` at the end of attributes which should not be null.

Reply:
You're right, will update the code.